src/lib/network/network_autoyast.rb

```diff
@@ -144,7 +144,7 @@ def activate_s390_devices(section = nil)
     def configure_hosts(write: false)
       log.info("NetworkAutoYast: Hosts configuration")
 
-      unless ay_current_profile.key? "host"
+      if ay_current_profile.empty?
```

Comment:
```suggestion
      if ay_host_section.empty?
```

Reply:
yes, it should just check host section. In fact change should be valid as it do some tricks with hosts config, but if section is empty it should be asme as if it is not there at all, not?

Reply:
yep, that is what I would expect